lnwire/reply_channel_range.go

```diff
@@ -46,7 +46,12 @@ func (c *ReplyChannelRange) Decode(r io.Reader, pver uint32) error {
 		return err
 	}
 
-	c.EncodingType, c.ShortChanIDs, err = decodeShortChanIDs(r)
+	// special handling to avoid error deep compare
```

Comment:
This has been fixed in the latest master.

Reply:
Nop, this was only partially fixed (for line 669).

If you add 
`numChanIDs = 0`

after line 701 at lnwire_test.go (so you force the random number to bu zero)  to make it:


```
			numChanIDs := rand.Int31n(5000)
			numChanIDs = 0

			req.ShortChanIDs = make([]ShortChannelID, numChanIDs)
```

and run the test you will see the problem.